src/components/Nav/index.js

```diff
@@ -32,7 +33,13 @@ function Nav() {
             </chakra.span>
           </HStack>
         </Link>
-        <Box>
+        <Flex alignItems="center" gridGap={variant}>
```

Comment:
We should switch this to `HStack` and use `spacing` because Chakra [technically doesn't support flex `gap` yet](https://github.com/chakra-ui/chakra-ui/issues/2720):

```suggestion
        <HStack spacing={{base: 3, sm: 10}}>
```

Since `HStack` aligns items to `center` by default, we also don't need to specify `alignItems`